services/user-sessions/src/user_sessions/serve.clj

```diff
@@ -1,63 +0,0 @@
-(ns user-sessions.serve
-  (:use [kameleon.user-session-queries]
```

Comment:
If they're not used elsewhere, we can probably delete some stuff out of this kameleon namespace, which we might as well clean out as we rewrite stuff.